Cargo.toml

```diff
@@ -11,7 +11,7 @@ description = "ListenBrainz API bindings for Rust"
 keywords = ["listenbrainz", "api", "bindings"]
 categories = ["api-bindings"]
 publish = true
-rust-version = "1.58"
+rust-version = "1.63.0"
```

Comment:
For some reason cargo-msrv on my machine claims the MSRV is 1.58, but the workflow claims 1.63.0. Not sure why this is happening.

Reply:
Is `cargo-msrv` up to date, as well as your Rsut toolchian? What does `rustc -V` report in the project dir?

Reply:
Yep. I updated Rust and cargo-msrv and ran it, and it spit out 1.58.1. However, for some reason now it's claiming 1.63.0, so I guess I may have had a stale build cache or something.